src/minify/minify-js.js

```diff
@@ -35,6 +35,8 @@ function minify(options) {
 
 function generateSyntaxTree(js, filePath) {
 
+    uglify.base54.reset();

```

Comment:
It probably would, or at least could, have. This hasn't been merged into the monolith yet, though.